CONTRIBUTING.md

```diff
@@ -33,6 +33,24 @@ information on using pull requests.
 This project follows [Google's Open Source Community
 Guidelines](https://opensource.google.com/conduct/).
 
+## Git Workflow
+
+This project conforms to the [Gitflow Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-wokflow).
+The repo contains two important branches: the `main` branch stores the official release history, and the `develop` branch contains
+the current working codebase.
+
+As a contributor, your workflow will be to create a temporary branch off of `develop`, add your code, and create a PR back to `develop`.
+Periodically, we will coordinate to publish release PRs to merge the latest state of `develop` back into `main`.
+
+
+## Conventional Commits
+
+This project expects PR titles to conform to the [conventional commits standard](https://www.conventionalcommits.org/en/v1.0.0/).
+In brief, your PR titles should be prefixes with `feat:`, `fix:`, `chore:`, `docs:`, or `test:`, to make it easy to parse for change logs.
+A bot will enforce this as part of our PR checks.
```

Comment:
It's not part of this PR, right?

Reply:
No, but we don't need a PR for this one, so I included the CONTRIBUTING update here